templates/zerver/base.html

```diff
@@ -16,8 +16,33 @@
                 {% endif %}
             {% endif %}
         {% endblock %}
+        <!-- The order of the alternate_icon and the icon here is
+        important.
+
+        The spec (https://html.spec.whatwg.org/multipage/links.html#rel-icon)
+        mentions "If there are multiple equally appropriate icons,
+        user agents must use the last one declared in tree order at
+        the time that the user agent collected the list of icons."
+
+        If the order was reversed and the user had
+        realm_icon_as_favicon set to true, we would have a bug
+        whereby after once getting to 0 unreads (ie this would not be
+        seen on first page load) the png image would be favoured in
+        place of the svg image ie the count would not be shown.
+
+        It's unclear why the same bug isn't seen with the default
+        zulip svg.
+
+        Also, the above bug may be chrome specific, (see:
+        https://crbug.com/1140920)
```

Comment:
https://crbug.com/1140920 isn’t related to alternate icon selection. Did you mean to paste a different link?

Reply:
Correct, I meant to link https://crbug.com/1162276, but the confusing thing is that the bug report describes the opposite of what we're seeing. It says that "with sizes="any" the .png is always chosen" what we're seeing is that "with sizes="any", the icon is chosen based on last declared order" and "without sizes="any", the .png is always chosen".